slides/en/net/merger/html-to-html/_index.md

```diff
@@ -0,0 +1,97 @@
+---
+title:  Merge HTML Files in C#
+url: /net/merger/html-to-html
+keywords: Merge HTML, HTML to HTML, Join HTML, Combine HTML, C# API, .NET Library
+description: Merge HTML to HTML in C#. Use .NET library API to combine HTML files
+---
+
+{{< blocks/products/pf/main-wrap-class isAutogenPage="true" >}}
+{{< blocks/products/pf/feature-page-wrap >}}
+
+{{< blocks/products/pf/feature-page-header h1="Merge HTML in C#" h2="Powerful cross-platform .NET API for merging HTML files using C# code on NET Framework, .NET Core, Windows Azure, Mono or Xamarin Platforms" >}}
+
+{{% blocks/products/pf/feature-page-section h2="Merge HTML to HTML using Aspose.Slides" %}}
+
+[**Aspose.Slides for .NET**](https://products.aspose.com/slides/net/) is a powerful .NET library used to create, convert, merge, and manipulate presentations, HTML files, and other documents. When you merge HTML to HTML, you are effectively combining the contents from 2 documents to obtain one HTML file. Aspose.Slides allows you merge HTML docs in different ways. You get to merge HTML with all their texts, formatting,styles, etc.
+
+{{% /blocks/products/pf/feature-page-section %}}
+
+
+
+
+{{% blocks/products/pf/feature-page-section  h2="Merge HTML to HTML in C#" %}}
+Using [**Aspose.Slides for .NET**](https://products.aspose.com/slides/net/), you can merge HTML files quickly with just a few lines of code
+
+{{% blocks/products/pf/agp/code-block title="C# code for merging HTML to HTML" offSpacer="true" %}}
+```cs
+using (Presentation pres = new Presentation())
```

Comment:
Please remove extra tabs. It should be:
```
using (Presentation pres = new Presentation())
{
    pres.Slides.RemoveAt(0); // remove default empty slide
    pres.Slides.AddFromHtml("page_1.html");
    pres.Slides.AddFromHtml("page_2.html");
    pres.Save("MergedFile.html", SaveFormat.Html);
}
```

Reply:
it should be fixed for all code snippets

Reply:
I've made changes to the new pages to resolve the issues.